cli/run_util.go

```diff
@@ -817,3 +826,61 @@ func runWorkflowWithConfiguration(
 	}
 	return buildRunResults, nil
 }
+
+func sendAnonymizedAnalytics(buildRunResults models.BuildRunResultsModel) error {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
AFAIK this does nothing, recover only recovers panics started **from** this function